src/genkeys.jl

```diff
@@ -56,9 +56,16 @@ LS0tLS1CRUdJTiBSU0EgUFJJVkFURSBLRVktLS0tLQpNSUlFb3dJQkFBS0NBUUVBNnpiRkdXQVZpYlIy
 ```
 """
 function genkeys(; user="\$USER", repo="\$REPO")
-    # Error checking. Do the required programs exist?
-    success(`which which`)      || error("'which' not found.")
-    success(`which ssh-keygen`) || error("'ssh-keygen' not found.")
+    if Sys.iswindows()
+        # Error checking. Do the required programs exist?
+        success(`where where`)      || error("'where' not found.")
+        success(`where git`)        || error("'git' not found.")
```

Comment:
This should be `success(`which ssh-keygen`) || error("'ssh-keygen' not found.")`, not Git?

Reply:
Fixed in https://github.com/JuliaDocs/DocumenterTools.jl/pull/33/commits/49ab299bce8824ff1139fa665c8e72ebb27ae2fa

Reply:
e30962b modified the wrong `genkeys` method I'm afraid.

Reply:
No, I check the existence of ssh-kegen both on Linux and Windows.
The tests pass if you see.

Reply:
This method does not use `git` though, so we shouldn't require it to be present here.

```suggestion
```

Reply:
done in https://github.com/JuliaDocs/DocumenterTools.jl/pull/33/commits/3fa8676703dc19227b8fe993520e4b26a4d4afad